docs/network-policy-recommendation.md

```diff
@@ -0,0 +1,148 @@
+# Network Policy Recommendation
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Deployment](#deployment)
+- [Perform NetworkPolicy Recommendation](#perform-networkpolicy-recommendation)
+  - [Run a policy recommendation job](#run-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Retrieve the result of a policy recommendation job](#retrieve-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration to secure Kubernetes
+network and applications. It analyzes the network flows collected by
+[Grafana Flow Collector](network-flow-visibility.md#grafana-flow-collector) to generate
+[Kubernetes NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+or [Antrea NetworkPolicies](https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
+The goal of this feature is to aid the cluster admin or developers to secure
+their applications based on the idea of the Zero Trust model.
+
+## Deployment
+
+To use the NetworkPolicy recommendation feature, please follow this [doc](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to deploy Flow Exporter and Flow Aggregator first.
+Please update the `#podLabels: false` to `podLabels: true` inside
+[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1),
+because policy recommendation feature needs Pod labels information to recommend network policies.
+
+Then, please follow these [deployment steps](network-flow-visibility.md#deployment-steps)
+to deploy the Grafana Flow Collector.
+
+After we finish the deployment of Grafana Flow Collector and can see some flow records in the Grafana UI,
```

Comment:
Is it required that we verify flows are visibile in Grafana?

Otherwise I'd just write: "Once Theia is deployed, the [Kubernetes Operator for Apache Spark](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator) will need to be installed in the cluster.

Reply:
Comment out these lines now since user won't need to deploy Spark Operator separately after PR#31 merged.